### PR TITLE
feat: add cocker support

### LINKED ISSUE #1: [new feature] docker support for selfhosting?
Hey! Could you add a dockerfile for it so you can selfhost using docker?

### PR DESCRIPTION
**Pull request by detarkende**
This adds a Dockerfile to the repo and updates the readme.

This should fix #1 .

You can test the solution with the published images from docker hub: https://hub.docker.com/r/detarkende/stremio-ncore-addon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,59 @@ If you do not own an Android TV / Chromecast / Firestick / Android phone or simi
 
 **Client**: the device where you will watch the content through the Stremio app.
 
+### Using docker
+
+Example docker run command:
+
+```sh
+docker run -d \
+    --name=transmission \
```
Comment (hauserkristof) [inline README.md:93]: We should change `transmission` to `streamio-ncore-addon` or something more descriptive

**Review by hauserkristof — APPROVED** [review]
LGTM!
One minor propose.

I'm going to test it properly in the afternoon

File: README.md (hunk 1 above)
Comment (detarkende) [reply README.md]: Shoot, that's a good catch, thanks!

I copied the command from the [linuxserver/transmission](https://hub.docker.com/r/linuxserver/transmission) repo so I must have left it there 😄 

Should be fixed now.

Comment (hauserkristof) [reply README.md]: also the PR's name says __c__ocker support instead of __d__cker support 😅

**Review by hauserkristof — APPROVED** [review]

**Comment by detarkende** [comment]
@hauserkristof you said you will test it later, but you approved. Can I merge, or should I wait?

**Comment by hauserkristof** [comment]
I think you can merge it. If I find anything strange during testing, I'm going to create an issue about it

**Pull request merged**